pkg/process/checks/process.go

```diff
@@ -356,6 +361,16 @@ func formatMemory(fp *procutil.Stats) *model.MemoryStat {
 	return ms
 }
 
+func formatNetworks(pid int32, conns map[int32][]*model.Connection, interval int) *model.ProcessNetworks {
```

Comment:
Minor: I think you could just take in a `[]*model.Connection` parameter, since you don't use `conns` or `pid` otherwise

Reply:
Yea I would change this, thanks!